### PR TITLE
update text  & links of presentations list

### PR DESCRIPTION
**Pull request by dmylonas**

### CODE REVIEW

File: colournaming/templates/index.html [hunk 1]
```diff
@@ -169,26 +168,8 @@ <h2 id="research">
...
-                <li>We will ask you to give us information on your cultural background.</li>
-            </ul>
-        </div>
-        <div class="padded">
             <h3>Crowdfunding Project:</h3>
-            <p>In September 2016, we've run a successful Kickstarter project, entitled <a href="https://www.kickstarter.com/projects/111984715/colours-of-babel">Colours of Babel</a>, to upgrade our online experiment to run on all new devises, minimize security threats and maintain our online presence free without advertisements. We thank all our 96 supporters for helping us raise 112% of our funding goal. You can find below a colour-coded list with our most precious donors:</p>
+            <p>In September 2016, we run a successful Kickstarter project, entitled <a href="https://www.kickstarter.com/projects/111984715/colours-of-babel">Colours of Babel</a>, to upgrade our online experiment to run on all new devises, minimize security threats and maintain our online presence free without advertisements. We thank all our 96 supporters for helping us raise 112% of our funding goal. You can find below a colour-coded list with our most precious donors:</p>
```
Comment (jstutters) [inline colournaming/templates/index.html]: Couple of typos:
"we run a successful Kickstarter" should be "we ran a successful Kickstarter"
"devises" should be "devices"

File: colournaming/templates/index.html [hunk 2]
```diff
@@ -258,13 +239,14 @@ <h2 id="research">
                 <li>Mylonas, D. (2015). The role of psychological primaries in cognitive colour spaces: a cross cultural colour naming study. In Colour Vision Meeting 2015, Colour Group GB.</li>
                 <li>Mylonas, D., Andrews, J., Griffin L.D. (2016). Variability in free colour naming across the full colour gamut. In AVA Christmas Meeting, London, UK.</li>
                 <li>Griffin L.D., Mylonas, D. (2016). Why so much talk about “red”? In AVA Christmas Meeting, London, UK.</li>
-                <li>Mylonas, D., MacDonald, L. (2017). Colour Naming for Colour Design. In Colour Design, ed Best, J. Elsevier</li>
+                <li>Mylonas, D., MacDonald, L. (2017). Colour Naming for Colour Design. <a href="https://www.elsevier.com/books/colour-design/best/978-0-08-101270-3"> In Colour Design</a>, ed Best, J. Elsevier</li>
```
Comment (jstutters) [inline colournaming/templates/index.html]: Remove leading space from inside <a></a> block ("270-3"> In Colour")

File: colournaming/templates/index.html [hunk 3]
```diff
@@ -258,13 +239,14 @@ <h2 id="research">
...
                 <li>Mylonas D. (2017) Features of GREEN colour categories across languages. Talk at The Colour Green meeting, Northern Chapter, Colour Group GB, Newcastle, UK.</li>
                 <li>Mylonas, D., MacDonald L., Griffin, L.D. (2017). Differences in Color Naming between British and American English Speakers. In 13th AIC Congress, South Korea.</li>
                 <li>Griber Y., Paramei G.V., Mylonas, D. (2017). Gender Differences in Russian Colour Naming. In 13th AIC Congress, South Korea.</li>
                 <li>Ulusoy B., Griffin L.D., Mylonas, D. (2017). Turkish Colour Naming on the Net. In 13th AIC Congress, South Korea.</li>
-                <li>Paramei G, Gibber Y., Mylonas D. (2017) An online colour naming experiment in Russian using Munsell colour samples, Color Research &amp; Application (in press), DOI:10.1002/col.22190.</li>
+                <li>Paramei G, Gibber Y., Mylonas D. (2017) An online colour naming experiment in Russian using Munsell colour samples, Color Research &amp; Application (in press), <a href="https://onlinelibrary.wiley.com/doi/10.1002/col.22190/full"> DOI:10.1002/col.22190.</a></li>
```
Comment (jstutters) [inline colournaming/templates/index.html]: Remove leading space from link text `22190/full"> DOI:10.1002/co`

File: colournaming/templates/index.html [hunk 4]
```diff
@@ -296,7 +278,7 @@ <h2 id="contact">
         <a class="github social-link" href="https://github.com/colournaming/colournaming"><!-- GitHub Logo --></a>
         <a class="email social-link" href="mailto:dmilonas@gmail.com"></a>
     </div>
-    <p class="lined-text">Made in London (and Berlin) with lots of love.</p>
+    <p class="lined-text">Made in London & Berlin</p>
```
Comment (jstutters) [inline colournaming/templates/index.html]: Replace ampersand with `&amp`.

**Comment by dmylonas** [comment]
ok

File: colournaming/templates/index.html [hunk 5]
```diff
@@ -278,7 +278,7 @@ <h2 id="contact">
         <a class="github social-link" href="https://github.com/colournaming/colournaming"><!-- GitHub Logo --></a>
         <a class="email social-link" href="mailto:dmilonas@gmail.com"></a>
     </div>
-    <p class="lined-text">Made in London & Berlin</p>
+    <p class="lined-text">Made in London &amp Berlin</p>
```
Comment (jstutters) [inline colournaming/templates/index.html]: Still missing a semicolon it should be `&amp;` ([see here for a guide](https://www.w3schools.com/html/html_entities.asp)).

**Pull request merged**